portfolio/src/main/java/com/google/sps/servlets/FavoriteColors.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that shows my favorite colors */
+@WebServlet("/Favorite-Colors")
+
+public class FavoriteColors extends HttpServlet {
+
+  @Override
+
+  
```

Comment:
Delete these two extra lines - the `@Override` should be directly over the function it overrides

```
@Override
public void doGet(...
```